admin/pg12/pg-9.5-stopped.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.1'
+
```

Comment:
Thanks for having added it to `admin/pg12/`, so that it is correctly ignored by `admin/configure list`.